src/main/java/com/gluonhq/NativeBaseMojo.java

```diff
@@ -100,6 +101,9 @@
     @Parameter(property = "client.nativeImageArgs")
     List<String> nativeImageArgs;
 
+    @Parameter(property = "client.linkFlags")
+    Map<String, TargetLinkFlagParameter> linkFlags;
```

Comment:
I tried `Map<String, List<String>>` but it wasn't working. Let me know if there's a way to do that in Maven.